### DISCUSSION TITLE
Configuration is not matching Astro Blog scheme

### DISCUSSION THREAD

**Discussion by rafay99-epic** [tinacms/tinacms#4849] [Help]
# Issue

I am developing a project using TinaCMS with Astro and adding a feature for multiple authors on my blog website. The desired structure for the author information in the blog post looks like this:

```yaml
author:
  name: "Abdul Rafay"
  image: "/author.jpg"
```

I want to incorporate this into TinaCMS via Git. Below is my current TinaCMS configuration:

```typescript
import { defineConfig } from "tinacms";

const branch = "main";

export default defineConfig({
  branch,
  clientId: "",
  token: "",
  build: {
    outputFolder: "admin",
    publicFolder: "public",
  },
  media: {
    tina: {
      mediaRoot: "",
      publicFolder: "public",
    },
  },
  schema: {
    collections: [
      {
        name: "post",
        label: "Posts",
        path: "/src/content/blog",
        fields: [
          {
            type: "string",
            name: "title",
            label: "Title",
            isTitle: true,
            required: true,
          },
          {
            type: "string",
            name: "description",
            label: "Description",
            required: true,
          },
          {
            type: "datetime",
            name: "pubDate",
            label: "Published Date",
            required: true,
          },
          {
            type: "boolean",
            name: "draft",
            label: "Draft",
            required: true,
          },
          {
            type: "image",
            name: "heroImage",
            label: "Hero Image",
            required: true,
          },
          {
            type: "object",
            name: "author",
            label: "Author",
            fields: [
              {
                type: "string",
                name: "name",
                label: "Author Name",
                required: true,
              },
              {
                type: "image",
                name: "image",
                label: "Author Image",
                required: true,
              },
            ],
          },
          {
            type: "rich-text",
            name: "body",
            label: "Body",
            isBody: true,
          },
        ],
      },
    ],
  },
  search: {
    tina: {
      indexerToken: "",
      stopwordLanguages: ["eng"],
    },
    indexBatchSize: 100,
    maxSearchIndexFieldLength: 100,
  },
});
```

However, when I launch TinaCMS and try to add an author, I encounter the following error:

![Error Image](https://github.com/user-attachments/assets/ef69b4ff-2f50-4e38-a273-ce7a7ef97fe7)

I am unsure what's causing this issue. Could you provide any suggestions for resolving it?

here is the project Link https://github.com/rafay99-epic/Astro-Portfolio-Blog/tree/Multiple-Author

**Comment by Klrfl** [comment]
I don't know if this is related to the error or not, but if you want multiple authors you should add `list: true` to the `authors` field.

**Reply by rafay99-epic** [reply]
> I don't know if this is related to the error or not, but if you want multiple authors you should add `list: true` to the `authors` field.


A list or array will be acceptable if there is more than one author, even if there is one author, but an author is an object and it has two attributes. 
1. Author Name 
2. Author Author Image 

In the Astro blog on which I am working, having an object is more easy to deal as compared to an array, so I was having an issue with Tina CMS, which does not identify  

```
author:
  name: "Abdul Rafay"
  image: "/author.jpg"
```

in the meta data of the markdown file. Like an object that has two attributes, name and image, if I create an array of an object, will Tina CMS be able to identify values?

**Reply by Klrfl** [reply]
well you did say you want to insert `multiple authors`, not an `author` so an array is perfectly fine. Also maybe the error has something to do with the required inputs? I'm not too sure

**Reply by rafay99-epic** [reply]
It could be, that is a good solution for the multiple authors, but that's for the responsive; if I find any solution, I will post the answers. Thanks for clearing things up.
